### ISSUE TITLE
Mods do not load with hardlinked FO4

### ISSUE THREAD

**Issue by KiPhemyst** [TanninOne/modorganizer#842]
Hello,

I have the following setup:
C:\Hardlinked\Fallout 4 <- Actual FO4 folder (SSD)
D:\Program Files\Steam\steamapps\common\Fallout 4 -> Hard link to FO4 (HDD)
D:\Games\Fallout 4 Mod Organizer <- Mod Organizer folder

When I run the game, no mods are loaded if I use F4SE, and if I run just Fallout4.exe it loads only interface mods, no .esp or .esm mods are loaded as far as I can tell.

MO Version: 2.0.7 rev 8d04c77

I have same exact setup for Skyrim with MO: 1.3.11 rev d1c81666578e+ and it works fine.

I tried running MO as admin, changing permissions on folders - it made no difference.

[usvfs-2016-06-26_20-01-12.log.txt](https://github.com/TanninOne/modorganizer/files/333797/usvfs-2016-06-26_20-01-12.log.txt)

**Comment by KiPhemyst** [comment]
Few more things - I "upgraded" my Skyrim to 2.0.7 and it also does not work, the game does not even start. And since I have a ton of mods for Skyrim, the log file produced is 40mb in size, however it does look similar to the FO4 log file with many repeated lines.

I'm running Windows 10 Pro x64 by the way.

**Comment by bryanlyon** [comment]
You need to have the mods on the same drive as your installation for VFS to work properly.  Not sure if hardlink would cause a problem in addition.  I suggest using Steam's built in libraries feature to install on the correct drive instead of using hardlinks.

**Comment by KiPhemyst** [comment]
> You need to have the mods on the same drive as your installation for VFS to work properly.

I thought MO 1.3 uses VFS and "I have same exact setup for Skyrim with MO: 1.3.11and it works fine."

I also said: "if I run just Fallout4.exe it loads only interface mods" so it is correctly placing 'Interface' folder from 'D:\Games\Fallout 4 Mod Organizer\mods\DEF_UI' to Fallout4 folder. Therefore it should work with other files as well.

Steam's built it libraries don't have "Move" button that would allow me to simply move the games from one drive to another without reinstalling the whole game (if it is very well hidden somewhere, let me know).

**Comment by KiPhemyst** [comment]
> never

except it is working right now and has been working for ages on two different machines. Here is part of a log from MO 1.3. I ran TES5Edit and told it to load Skyrim.esm and a random mod:

`INFO (21:56:49.0206): injecting to D:\Games\Skyrim Tools\TES5Edit 3.0.32 exp\TES5Edit.exe`
`INFO (21:56:49.0206): using profile Skyrim Main`
`INFO (21:56:49.0207): data path is D:\Program Files\Steam\steamapps\common\Skyrim\data` <- Steam library folder on D:\
`INFO (21:56:49.0207): data path may also be a junction to C:\Hardlinked\Skyrim\Data` <- See, it knows that Skyrim is actually "a junction" from C:\ drive
`DEBUG (21:56:49.0210): mods are in "D:\Games\Skyrim Mod Organizer\mods"` <- and mods are in D:\

Therefore it must be a bug or a missing feature in MO 2.0

**Comment by bryanlyon** [comment]
Yes, it detected the junction in your 1.3 and it's using the same drive.  However, MO does not change the Fallout 4 binaries, and those may not work so well with junctions.  My advice stands of removing the junction and ensuring everything is on the same drive.  If you do that and keep experiencing problems then please come back and let us know about that, but without removing the junction, there is not much else we can do.
